services/movie_session.py

```diff
@@ -0,0 +1,43 @@
+from db.models import MovieSession, Movie, CinemaHall
+
+
+def create_movie_session(
+        movie_show_time: str,
+        movie_id: int,
+        cinema_hall_id: int,
+) -> MovieSession:
+    return MovieSession.objects.create(
+        show_time=movie_show_time,
+        movie=Movie.objects.get(id=movie_id),
+        cinema_hall=CinemaHall.objects.get(id=cinema_hall_id),
+    )
+
+
+def get_movies_sessions(session_date: str = None) -> list[MovieSession]:
+    if session_date is not None:
+        return MovieSession.objects.filter(show_time__date=session_date)
+    return MovieSession.objects.all()
+
+
+def get_movie_session_by_id(movie_session_id: int) -> MovieSession:
+    return MovieSession.objects.get(id=movie_session_id)
+
+
+def update_movie_session(
+        session_id: int,
+        show_time: str = None,
+        movie_id: int = None,
+        cinema_hall_id: int = None,
+) -> None:
+    movie_session = MovieSession.objects.get(id=session_id)
+    if show_time is not None:
+        movie_session.show_time = show_time
+    if movie_id is not None:
+        movie_session.movie = Movie.objects.get(id=movie_id)

```

Comment:
Why you need to get object when you can use movie_id = movied_id
CinemaHall have the same problem